swift/README.md

```diff
@@ -1,7 +1,7 @@
 # TBD Swift Examples
 
-These examples use TBD's Swift libraries ([tbdex-swift](https://github.com/TBD54566975/tbdex-swift) or [web5-swift](https://github.com/TBD54566975/web5-swift)).
+These examples use TBD's Swift libraries ([web5-swift](https://github.com/TBD54566975/web5-swift) and sometimes [tbdex-swift](https://github.com/TBD54566975/tbdex-swift)).
```

Comment:
can we remove the possessive `TBD's`? all instances. they are open source libraries and dont belong to TBD

Reply:
What term should we use to describe these libraries that doesn't include all other random dependencies? is "web5" better?